command/harness/setup.go

```diff
@@ -218,6 +219,15 @@ func handleSetup(ctx context.Context, logr *logrus.Entry, r *SetupVMRequest, env
 		WithField("id", instance.ID).
 		WithField("instance_name", instance.Name)
 
+	// Since we are enabling Hardware acceleration for GCP VMs so adding this log for GCP VMs only. Might be changed later.
+	if instance.Provider == types.Google {
+		var isHardwareAccelerationEnabled = "disabled"
+		if r.EnableNestedVirtualization {
+			isHardwareAccelerationEnabled = "enabled"
+		}
+		logr.Traceln(fmt.Sprintf("creating VM instance with hardware acceleration %s", isHardwareAccelerationEnabled))
```

Comment:
This much should be enough here
logr.Traceln(fmt.Sprintf("creating VM instance with hardware acceleration as %s", r.EnableNestedVirtualization))